content/open-tech-will-save-us/index.md

```diff
@@ -0,0 +1,14 @@
++++
+title = "Open Tech Will Save Us"
+template = "open-tech-will-save-us.html" 
++++
+
+[Join the live chat now!](https://matrix.to/#/!AnacGSwlCZcUuAfcEU:matrix.org?via=matrix.org&via=bpulse.org&via=uhoreg.ca)
```

Comment:
owtsu has many aliases, why is this using the room ID (which will go out of date if the room ever upgrades)?